### PR TITLE
Added dark mode

### PR DESCRIPTION
**Pull request by jokenetwork**
Almost all new browsers support dark mode, which is why I implemented it.

### CODE REVIEW

File: style.scss [hunk 1]
```diff
@@ -30,6 +30,24 @@ Underscores is distributed under the terms of the GNU GPL v2 or later.
   --transition-duration: 333ms;
 }
 
+figcaption a{color: #FFF!important;}
```
Comment (prtksxna) [inline style.scss]: Should this be inside the media query?

Comment (jokenetwork) [reply style.scss]: Yes, I think it should've been in the media query, don't know how I messed this up.
Probably we should set the line-height to something like 1.9.

**Comment by jokenetwork** [comment]
It was a pleasure! I hope it's alright now.

**Comment by prtksxna** [comment]
I'll deploy this to [wordpress.org](https://wordpress.org/themes/zuari/) after making a few minor changes (mostly linting related). Thanks again for working on this @jokenetwork 🙏🏽

**Comment by jokenetwork** [comment]
Can't wait to update my theme!

**Comment by prtksxna** [comment]
I think I'll have to make this a customizer setting. This might mess up the view for people who have custom colors set up 😓

**Comment by jokenetwork** [comment]
Is there an option that users can change the colors of the theme entirely?
If yes, could you just add "Dark-Mode colors" to it? Might solve this.

**Comment by prtksxna** [comment]
> Is there an option that users can change the colors of the theme entirely?

Yeah, there is an option under Customize > Colors. 

> If yes, could you just add "Dark-Mode colors" to it? Might solve this.

Exactly what I was thinking. I found some bugs in my code around this, so will fix those too.

**Pull request merged**